steely/plugins/define.py

```diff
@@ -42,18 +44,39 @@ def main(bot, author_id, message, thread_id, thread_type, **kwargs):
             bot.sendMessage(ANGRY_STRING, thread_id=thread_id,
                             thread_type=thread_type)
             return
+    elif command == 'image':
+        if ' ' in text:
+            command, image_url = text.split(' ', 1)
+            try:
+                bot.sendRemoteImage(
+                    image_url, thread_id=thread_id, thread_type=thread_type)
+                text = None
+            except Exception:
+                bot.sendMessage(BAD_MEME_STRING,
+                                thread_id=thread_id, thread_type=thread_type)
+        else:
+            bot.sendMessage(ANGRY_STRING, thread_id=thread_id,
+                            thread_type=thread_type)
     if not command.startswith('~'):
         command = f'~{command}'
     if len(command) > LIMIT:
         bot.sendMessage("that's too long dickhead",
                         thread_id=thread_id, thread_type=thread_type)
         return
     search = CMD_DB.get(CMD.cmd == command)
+
+    data = {}
+    if image_url is not None:
+        data["image"] = image_url
```

Comment:
So here are you removing the old text? What is the state of the db after creating a piece of text and then defining an image

Reply:
ooh good question.

I think each ~word should only refer to either an image or a sentence or a code, just like overwriting words with code now deletes the words.